README.md

```diff
@@ -42,7 +42,8 @@ Prerequisites:
 - [docker and docker-compose](http://docs.docker.com/compose/install/)
 - [Notary server configuration](#configure-notary-server)
 
-As setup, lets build notary and then start up a local notary-server.
+As setup, lets build notary and then start up a local notary-server (don't forget
+to add `127.0.0.1 notary-server` to your `/etc/hosts`).
```

Comment:
Non-blocking: "or `<docker-machine IP> notary-server` if you are using docker-machine" maybe?  Probably overly pedantic.